src/src/components/Molecules/README.md

```diff
@@ -0,0 +1,5 @@
+# Molecules
```

Comment:
What is Molecules?

Reply:
I have a link to the Atomic Design principle in that file - https://github.com/maxxxxxdlp/calendar-stats/tree/first-steps/src/src/components/Molecules

TLDR: `Atoms` are the most basic building blocks of the UI. `Molecules` are a bit higher level than that and can combine multiple `Atoms`.

Reply:
Interesting - seems cool!